front/src/modules/scenario/components/AddOrEditScenarioModal.tsx

```diff
@@ -47,14 +47,17 @@ export default function AddOrEditScenarioModal({
   };
 
   const [deleteScenarioRTK] =
```

Comment:
you can remove the 'RTK' here

Reply:
I can but had to change the deleteScenario event function name